### ISSUE TITLE
Off-Canvas contents invisible in Safari on iPhone 6S Plus in landscape mode, appears after scrolling.

### ISSUE THREAD

**Issue by N8Solutions** [gantry/gantry5#2036]
The Off-Canvas contents are invisible only in Safari, only on iPhone 6S Plus, and only in landscape mode, and they appear after scrolling. I can only reproduce this on a few sites I have worked on and I can't figure out the why behind this behaviour. It does not happen on the Helium demo page.

I have also tested the sites exhibiting this behaviour on an iPhone 6S as well as an iPhone 4S, and an older model Android, and the problem does not occur on any of them.

I have also used the Chrome & Firefox browsers on the iPhone 6S Plus and the problem doesn't exist. It only happens with the Safari browser on iPhone 6S Plus. It might happen on say an iPhone 7 Plus but I do not have one to test that on.

I have also tried reproducing the issue by resizing the browser window on my computer but it functions normally.

The issue is that on a few sites, with the iPhone 6S **Plus**, only in Safari, and only in landscape mode, **at the TOP of the page**, when you click on the **Mobile Menu Icon** the Off-Canvas section shows but the content is invisible, everything is white. If you perform a **tap-drag-release** from the bottom up over the menu area you as if you are scrolling down the page then the contents of the Off-Canvas section (i.e. the menu) show up. However, if you close the Off-Canvas Section and are back at the top of the page then the same thing happens again and again.

On some of the sites I have made the Mobile Menu Icon "Sticky" so it stays put. On those, if I scroll down the page a little, then try to open the Mobile Menu, it opens and everything is visible.

I would appreciate if some of you could test this behaviour out on your large screen phones, i.e. iPhone 6S Plus, iPhone 7 Plus.

Here is a list of the sites where the problem exists.
[Site 1](http://pcct.n8solutions.org)
[Site 2](https://www.n8solutions.com)
[Site 3](http://www.xeiropraktiki.gr)
[Site 4](https://www.idratherbequilting.com)
[Site 5](http://precisionchirovt.com)

Thanks in advance for the help!

**Comment by w00fz** [comment]
I tried on the Simulator the iPhone 6S Plus and iOS 10.3. With all the test sites you linked I cannot reproduce the issue and it seems to be working fine.
I have an iPhone 7 which also seems to work fine. This evening when my wife comes back from work I can test on her 7 Plus, at the moment I only have the Simulator at my disposition which seems to work just fine.

By the sound of it it looks like this is mostly a rendering issue. There are a few tricks you can try to workaround the rendering issues. One consists in adding a `transform: translateZ(0)` to the `#g-offcanvas`, alternatively you can try also backface visibility.

Here the complete rules:

```css
#g-offcanvas {
    transform: translateZ(0);
}
```
or

```css
#g-offcanvas {
    backface-visibility: hidden;
}
```

or even the two combined. Give it a try and as soon as I can I will try on the physical iPhone 7 Plus.

**Comment by N8Solutions** [comment]
@w00fz Thank you!
I tried implementing what you said however it did not work. I also looked at the code and noticed that the off-canvas section code is greyed out. In nucleus there is a `~~display: none~~;`. I've attached a screen shot for you to see what I'm referring to in the code. Thanks for looking in to this!

![offcanvas greyed out](https://user-images.githubusercontent.com/12389693/27707155-f3f9adc0-5d1c-11e7-9b58-7d13911227d4.jpg)
